_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+In a DevOps release process, 2 weeks of work takes a day or less to release. All the functions are encapsulated within one cross-functional team per product. Interaction with central enablement teams such as IT Security, Accessibility and Testing Services is through automated checks and processes incorporated into the build. These central operational teams are instead tasked with continually addressing bottlenecks and championing improvements rather than solely a fire fighting role. An incident response role still exists, things will go wrong sometimes regardless of any process or methodology taken. The outcome however, is not to add an unrealistic burden to operational staff to manually ensure something doesn't happen again. Problems are fixed at the root by improving the process and incorporating automated checks into build and development tools.
+
+<table>
+ <thead>
+    <tr>
+        <td>Item</td>
+        <td>Current Release</td>
+        <td>DevOps Releases</td>
+    </tr>
+ </thead>
+ <tbody>
+    <tr>
+        <td>Time to release</td>
+        <td>3-5 months.</td>
+        <td>Daily releases or more frequently.</td>
+    </tr>
+    <tr>
+        <td>Number of stakeholders</td>
+        <td>Thirteen individual stakeholders with distinct functional roles.</td>
+        <td>One cross-functional team focused on a product or project with all the required skillset to execute.</td>
+    </tr>
+    <tr>
+        <td>Compliance</td>
+        <td>Manual, tedious and error prone. Over burdens operational staff and causes a lot of waiting and toil for developers.</td>
+        <td>Automated with every build which reduces the risk of error. Frees up operational staff to focus on improvement and innovation. Developers have quick feedback.</td>
+    </tr>
+    <tr>
+        <td>Reporting and metrics</td>
+        <td>Manual collection of data, tedious and often inaccurate.</td>
+        <td>Automated data collection through telemetry from continuous integration and build pipelines. Wealth of information allows the creation of rich automated reports to identify issues and resolve quickly.</td>
+    </tr>
+
+ </tbody>
+</table>
+
+This drastically reduces lead time, we are no longer playing a game of catch up. We are able to put our focus on delivering the best possible services to our citizens rather than navigating a process. Developers are able to focus on their work of adding features and addressing technical debt. Operational staff are not over burdened by being constantly in fire fighting mode and can instead focus on innovating and improving the process. Management is able to quickly deliver and exceed their commitments. Risk is reduced by ensuring that every build of an application or service works as expected and is automatically compliant to our security and accessibility commitments that we are bound to by law.
```

Comment:
Similar to above, I think the comma should either be a semi colon or replaced with 'as'.

I think 'fire fighting mode' is hyphenated ...fire-fighting mode

Is it 'compliant to' or 'compliant with'? in the sentence '...works as expected and is automatically compliant to'